packages/pkp-base/src/lib/pkp-base.ts

```diff
@@ -222,6 +200,14 @@ export class PKPBase<T = PKPBaseDefaultParams> {
     }
   }
 
+  private validateAuthContext() {
+    if (this.controllerAuthSig && this.authContext) {
+      throw new Error(
```

Comment:
can we use `this.throwError` in this class instead? we probably need to clean up in other places as well in this file

Reply:
Changed